x-pack/plugin/core/template-resources/src/main/resources/traces@settings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "template": {
+    "settings": {
+      "index": {
+        "lifecycle": {
+          "name": "traces"
```

Comment:
lifecycle is missing

Reply:
The implication of this is that index management page will show a "policy has lifecycle errors" error, as well as the following message when clicking into it.

```
{
  "step_info": {
    "type": "illegal_argument_exception",
    "reason": "policy [traces] does not exist"
  }
}
```

Reply:
This is in core, but from what I see it's only used by `otel-data` - we decided to not set lifecycle for `otel-data` yet. And it seems `apm-data` sets lifecycles differently. So from what I see it's safe to remove this here. Basically a copy-paste leftover.

Reply:
Removed in https://github.com/elastic/elasticsearch/pull/111091/commits/3bb6737b99f9f3fa89312885607af66aef603904